test.config.js

```diff
@@ -1,8 +1,8 @@
-require('babel-register')
-require('ts-node').register()
+require('@babel/register')
+// require('ts-node').register()
 require('coffee-script/register')
 require('source-map-support/register')
```

Comment:
Is this still necessary? If all transpilation happens at runtime, don’t they then keep the source-maps around?

Reply:
Good question, let me remove and see what happens